app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class='container' >
+  <div class='loginForm-new-transaction'>
+
+      <h2>New Transactions</h2>
+       <br>
+      <%= form_with model: @transaction, url: group_transactions_path  do |f|%>
+
+        <div class="form_inputs">
+          <div>
+            <%= f.text_field :name, required: true, placeholder: "Name of New Transaction"%>
+          </div>
```

Comment:
- [ ] Clicking on the "Back" button (<), when on the "add new transaction page" the user should navigate to the transactions page for that category, but instead it takes the user to the categories page. Please fix this 👍